sdk/monitor/azquery/metrics_client_test.go

```diff
@@ -156,3 +159,31 @@ func TestNewListNamespacesPager_Failure(t *testing.T) {
 	}
 
 }
+
+func TestMetricsAPIVersion(t *testing.T) {
+	apiVersion := "2023-10-01"
```

Comment:
consider using a value that obviously isn't a real version so it's clear this test doesn't redundantly specify the default